port: /dev/ttyUSB1
baud: 115200
timeout: 0.1
rate: 50
sensorstate_rate: 10

useSonar: False
useImu: True
use_base_controller: True
base_controller_rate: 10

# For a robot that uses base_footprint, change base_frame to base_footprint
base_frame: base_footprint

# === Robot drivetrain parameters
wheel_diameter: 0.105
wheel_track: 0.33
encoder_resolution: 11 
gear_reduction: 172
motors_reversed: False

# === PID parameters
Kp: 2.0
Ki: 80
Kd: 0.0
Ko: 50
accel_limit: 1.0

useSmoother: True
usedebug: False